.github/copilot-instructions.md

```diff
@@ -29,6 +30,7 @@ Use these instructions for test generation as well.
 -   Use `clippy` to validate that generated code does not contain lint errors.
 -   If you have trouble generating safe, efficient, maintainable, and lint-free code, insert a `TODO` comment describing what should happen.
 -   All imported types, constants, functions, modules, and macros should be imported explicitly. Never import `*`.
+-   Avoid modifying generated code, found in `generated` subdirectories. These files are generated by external tools and should not be edited manually.
```

Comment:
I'd be more forthright:
```suggestion
-   Do not modify generated code, found in `generated` subdirectories. These files are generated by external tools and should not be edited manually.
```
Or was the idea that it could for temporary changes like this until the emitter catches up?